services/notify/src/notifier.js

```diff
@@ -117,7 +110,8 @@ class Notifier {
     });
   }
 
-  async irc({channel, user, message}) {
+  async irc(msg) {
```

Comment:
Could we just call this `messageRequest` or something? It will make the following function a bit less confusing.